src/Elastic.Apm/Api/IError.cs

```diff
@@ -7,6 +7,7 @@ namespace Elastic.Apm.Api
 	/// <summary>
 	/// Represents an error which was captured by the agent.
 	/// </summary>
+	[Specification("docs/spec/errors/error.json")]
```

Comment:
In https://github.com/elastic/apm-server/pull/4427 this got change. I adapted the path locally, but then I run into another problem (comment below with the line).

Reply:
I've updated to the v2 specs